hummingbot/connector/exchange/coinflex/coinflex_exchange.py

```diff
@@ -726,49 +721,23 @@ async def _user_stream_event_listener(self):
-                            fill_price = coinflex_utils.decimal_val_or_none(order_data.get("matchPrice"))
-                            exec_amt_quote = exec_amt_base * fill_price if exec_amt_base and fill_price else None
-                            fee_paid = coinflex_utils.decimal_val_or_none(order_data.get("fees"))
-                            if fee_paid:
-                                fee = TradeFeeBase.new_spot_fee(
-                                    fee_schema=self.trade_fee_schema(),
-                                    trade_type=tracked_order.trade_type,
-                                    percent_token=order_data.get("feeInstrumentId"),
-                                    flat_fees=[TokenAmount(amount=fee_paid, token=order_data.get("feeInstrumentId"))]
-                                )
-                            else:
-                                fee = self.get_fee(base_currency=tracked_order.base_asset,
-                                                   quote_currency=tracked_order.quote_asset,
-                                                   order_type=tracked_order.order_type,
-                                                   order_side=tracked_order.trade_type,
-                                                   amount=tracked_order.amount,
-                                                   price=tracked_order.price,
-                                                   is_maker=True)
-                            trade_update = TradeUpdate(
-                                trading_pair=tracked_order.trading_pair,
-                                trade_id=int(order_data["matchId"]),
-                                client_order_id=client_order_id,
-                                exchange_order_id=str(order_data["orderId"]),
-                                fill_timestamp=int(order_data["timestamp"]) * 1e-3,
-                                fill_price=fill_price,
-                                fill_base_amount=exec_amt_base,
-                                fill_quote_amount=exec_amt_quote,
-                                fee=fee,
-                            )
-                            self._order_tracker.process_trade_update(trade_update=trade_update)
-                        order_update = OrderUpdate(
-                            trading_pair=tracked_order.trading_pair,
-                            update_timestamp=int(order_data["timestamp"]) * 1e-3,
-                            new_state=CONSTANTS.ORDER_STATE[order_data["status"]],
-                            client_order_id=client_order_id,
-                            exchange_order_id=str(order_data["orderId"]),
-                        )
-                        self._order_tracker.process_order_update(order_update=order_update)
+                    except asyncio.TimeoutError:
+                        self.logger().error(f"Failed to get exchange order id for order: {tracked_order.client_order_id}")
+                        raise
```

Comment:
The exchange order id is not required here. It should not be requested.

Reply:
Please see above